sim/paladin/item_sets_pve.go

```diff
@@ -187,3 +187,47 @@ var ItemSetMercifulJudgement = core.NewItemSet(core.ItemSet{
 		},
 	},
 })
+
+var ItemSetRadiantJudgement = core.NewItemSet(core.ItemSet{
+	Name: "Radiant Judgement",
+	Bonuses: map[int32]core.ApplyEffect{
+		2: func(agent core.Agent) {
+			// 2 pieces: Increases damage done by your damaging Judgements by 20% and your Judgements no longer consume your Seals on the target.
+			paladin := agent.(PaladinAgent).GetPaladin()
+			core.MakePermanent(paladin.RegisterAura(core.Aura{
+				Label: "S03 - Item - T2 - Paladin - Retribution 2P Bonus",
+				OnReset: func(aura *core.Aura, sim *core.Simulation) {
```

Comment:
This could use `OnInit` so it will run once after initialiation and not every iteration

Reply:
That would also mean you don't have to check if the effect was applied. I think we should also try to keep that code here within the same `OnInit` callback

Reply:
fixed